Library/Homebrew/cask/lib/hbc/artifact/pkg.rb

```diff
@@ -61,17 +61,15 @@ def run_installer(pkg_description)
       end
 
       def with_choices_file(choices)
-        unless choices
-          yield nil
-          return
-        end
+        return yield nil unless choices
 
+        file = Tempfile.new(["choices", ".xml"])
```

Comment:
This can stay inside the block, as this doesn't change anything. Other than that everything is looking fine.